src/main/java/org/codehaus/mojo/versions/UseLatestVersionsMojo.java

```diff
@@ -115,11 +115,7 @@ protected void update( ModifiedPomXMLEventReader pom )
                 useLatestVersions( pom, list);
             }
         }
-        catch ( ArtifactMetadataRetrievalException e )
-        {
-            throw new MojoExecutionException( e.getMessage(), e );
-        }
-        catch ( IOException e )
+        catch ( ArtifactMetadataRetrievalException | IOException e )
```

Comment:
var name while at it?

Reply:
why? it;s just a simple exception and the variable is only available for one line the rethrow it within another exception